inventory/byo/hosts.ose.example

```diff
@@ -167,6 +167,14 @@ openshift_release=v3.6
 # modify image streams to point at that registry by setting the following to true
 #openshift_examples_modify_imagestreams=true
 
+# If oreg_url points to a registry requiring authentication, provide the following:
+oreg_host: example.com:443
+registry_auth_user: some_user
+registry_auth_token: 'my-token'
```

Comment:
I thought about this.  I feel it should actually be the other way around, but that's outside the scope of this PR.

I suppose this would also work in something like this:
oreg_url: somehost.com:443/somedir/registry/openshift3/{{ component }}/{{ version }}
oreg_host: somehost.com:443/somedir/registry

What's the best way to parse oreg_url to derive oreg_host in both situations?
Better situation is this:
oreg_host = somehost.example.com
oreg_url = {{ oreg_host }}/{{ oreg_suffix }}

But that would probably break existing users and would require some refactoring.